src/test/kialo.test.ts

```diff
@@ -0,0 +1,41 @@
+import { assertType, expect, test } from "vitest";
+import * as arguebuf from "../index.js";
+import * as model from "../model/index.js";
+import * as fs from "fs";
+
+const file_path = "src/test/kialoTestFile.txt";
```

Comment:
did you try to write `import file from "./kialoTestFile.txt"? If I remember correctly, vite should be able to do that. Then you would not need to read the file manually.

Reply:
Yes, I did. " import file from "./kialoTestFile.txt" " doesn't give you the content of the text file. It just gives you the file path. In this case: "/src/test/kialoTestFile.txt".

Reply:
I found the following documentation entry: https://vitejs.dev/guide/assets.html#importing-asset-as-string

Does that work? Would be interesting for my upcoming changes to the testing structure.

Reply:
Yes, it works now.
The "?raw" suffix was missing.

Reply:
Great, thank you! In principle, this PR can be merged once I have added the new way of importing the test files.

In the meantime, you could already investigate #14 and open a separate PR. This xAIF format is currently not implemented in the Python version of Arguebuf, so we later need to implement this feature in Python as well. You will have to define a new schema for this format, but if I remember correctly, the `AIF` key of xAIF directly corresponds to the "old" AIF format.